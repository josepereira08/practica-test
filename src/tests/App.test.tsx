import { describe, expect, test } from "vitest"
import { shallow } from "enzyme"
import App from "../App"

describe("Testing <App/>", () => {
  test("it should show correctly <App/> component", () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})
