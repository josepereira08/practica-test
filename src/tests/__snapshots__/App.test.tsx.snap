// Vitest Snapshot v1

exports[`Testing <App/> > it should show correctly <App/> component 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="/src/logo.svg"
    />
    <p>
      Hello Vite + React!
    </p>
    <p>
      <button
        onClick={[Function]}
        type="button"
      >
        count is: 
        0
      </button>
    </p>
    <p>
      Edit 
      <code>
        App.tsx
      </code>
       and save to test HMR updates.
    </p>
    <p>
      <a
        className="App-link"
        href="https://reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        Learn React
      </a>
       | 
      <a
        className="App-link"
        href="https://vitejs.dev/guide/features.html"
        rel="noopener noreferrer"
        target="_blank"
      >
        Vite Docs
      </a>
    </p>
  </header>
</div>
`;
